### DISCUSSION TITLE
New Feature about image_demo.py

### DISCUSSION THREAD

**Discussion by JiayuXu0** [open-mmlab/mmyolo#127] [Ideas]
For image_demo.py, I want to add these features.
1. Add the inference test and result output of the images folder, cameras, video streams, and asynchronous stream.
2. Add the switch of whether to save the result file
3. Add the switch of whether to save the inference result txt
4. Increase the adjustment of the iou threshold
5. Add a switch for whether to display the test results on the screen.

**Comment by RangeKing** [comment]
I think using `glob` syntax, like 'path/*.jpg', as an input source is very convenient.

**Reply by JiayuXu0** [reply]
Thanks for your suggestion.

**Comment by JiayuXu0** [comment]
For input like camera and video streams， it's better inference in serial. So maybe creating a new async_demo.py file is a better choice for the asynchronous stream.
